packages/material-ui/src/styles/createMuiTheme.js

```diff
@@ -69,11 +71,11 @@ function createMuiTheme(options = {}, ...args) {
                 'You can not override it like this: ',
                 JSON.stringify(node, null, 2),
                 '',
-                'Instead, you need to use the $ruleName syntax:',
+                `Instead, you need to use the '&.Mui-${key}' syntax:`,
```

Comment:
Didn't we have some helper that generates these? Or do we have the `Mui-` prefix hardcoded everywhere?

Reply:
`Mui` is hardcoded so far, I wonder if developers will ask to customize now that we do no longer have a global class name generator. It could be interesting for white-labeling purposes.

Reply:
Let me double check, based on the docs, this is how they look like, but would be nice to re-use utility if we have.

Reply:
Done with https://github.com/mui-org/material-ui/pull/24535/commits/b383f0509639f32fc3a14a7ce4c499a3f11d5f82